mdm/mdm/command.go

```diff
@@ -6,7 +6,8 @@ import (
 )
 
 type CommandRequest struct {
-	UDID string `json:"udid"`
+	UDID        string  `json:"udid"`
+	CommandUUID *string `json:"command_uuid,omitempty"`
```

Comment:
I assume because it's optional. In cases like this it's hard to decide between "unset" and "set as empty string". 

Having an empty string value would always be an error rather than a default, so we can use the empty value as optional. 

I'm fine with either keeping it as a pointer or using empty string to signal the optional. Whatever you go with, just leave a comment in the code.

Reply:
Yep, I picked a pointer so that it could be nil by default. I wasn't really sure what the "go-y" way to signal optionals is but found that nil seemed most explicit. Is there a better/standard way?

Regardless, I'll toss in a comment on the fixup. Thanks!

Reply:
Right on; I had an idea that was the idea. As @groob said because there isn't a valid case where we'd use an empty value (and so we'd arguably want to check for both an empty string as well as a nil/optional value) it seems simpler to use a `string` here even at the additional resource usage of an empty variable (which we have anyway, due to the new local var introduced). Also it matches the other struct members and makes the default assignment in the other suggestion easier as it doesn't need to be conditional or initialized first. That was my thinking anyway. :)